lib/lightning/cw_lightning.dart

```diff
@@ -0,0 +1,69 @@
+part of 'lightning.dart';
+
+class CWLightning extends Lightning {
+
+  @override
+  String formatterLightningAmountToString({required int amount}) =>
+      bitcoinAmountToString(amount: amount * 100000000);
+
+  @override
+  double formatterLightningAmountToDouble({required int amount}) =>
+      bitcoinAmountToDouble(amount: amount * 100000000);
+
+  @override
+  int formatterStringDoubleToLightningAmount(String amount) =>
+      stringDoubleToBitcoinAmount(amount * 100000000);
+
+  WalletService createLightningWalletService(
+      Box<WalletInfo> walletInfoSource, Box<UnspentCoinsInfo> unspentCoinSource) {
+    return LightningWalletService(walletInfoSource, unspentCoinSource);
+  }
+
+  @override
+  List<LightningReceivePageOption> getLightningReceivePageOptions() =>
+      LightningReceivePageOption.all;
+
+  @override
+  ReceivePageOption getOptionInvoice() => LightningReceivePageOption.lightningInvoice;
+
+  @override
+  ReceivePageOption getOptionOnchain() => LightningReceivePageOption.lightningOnchain;
+
+  @override
+  String satsToLightningString(int sats) {
+    const bitcoinAmountLength = 8;
+    const bitcoinAmountDivider = 100000000;
+    const lightningAmountDivider = 1;
+    final bitcoinAmountFormat = NumberFormat()
+      ..maximumFractionDigits = bitcoinAmountLength
+      ..minimumFractionDigits = 1;
+
+    String formattedAmount = bitcoinAmountFormat.format(sats);
+    return formattedAmount.substring(0, formattedAmount.length - 2);
```

Comment:
why -2?

Reply:
this removes the last 2 characters off of the string
i.e. "100.0" -> "100"